src/test/java/dev/openfeature/sdk/EvalContextTest.java

```diff
@@ -38,7 +40,7 @@ public class EvalContextTest {
 
         assertEquals(4, ec.getValue("int").asInteger());
 
-        assertEquals(dt, ec.getValue("dt").asInstant());
+        assertEquals(dt, ec.getValue("dt").asInstant().truncatedTo(ChronoUnit.MILLIS));
```

Comment:
I don't have a problem with this, but I'm curious... why was this needed? This doesn't seem to have anything to do with slf4j.